apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts

```diff
@@ -208,7 +217,7 @@ export class AdvanceReceiptAdjustmentComponent implements OnInit {
      * @memberof AdvanceReceiptAdjustmentComponent
      */
     public addNewBlankAdjustVoucherRow(): void {
-        if (this.adjustPayment.grandTotal - this.adjustPayment.totalAdjustedAmount >= 0) {
+        if (this.adjustPayment.grandTotal - this.adjustPayment.totalAdjustedAmount - this.depositAmount + this.adjustPayment.tdsTotal + this.adjustPayment.tcsTotal >= 0) {
```

Comment:
Consider moving this expression into a typescript getter that will help to reduce the expression size and make it more readable